src/migrations/20181206163000_move_out_bundles_metadata.js

```diff
@@ -41,7 +39,8 @@ export const up = async (db, config, logger) => {
   await deduplicateBundles(db);
   await db.collection('bundle_metadata').createIndex({bundleId : 1}, {unique: true});
 
-  if (await db.collection('bundles').indexExists(['bundleId_1'])) {
+  const collectionExists = (await db.listCollections({name: 'bundles'}).toArray()).length > 0;
+  if (collectionExists && await db.collection('bundles').indexExists(['bundleId_1'])) {
```

Comment:
We have a lot of duplication here. I think it's time for some `migration_helpers.js`